scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+create table doctors(
+  id serial primary key
```

Comment:
The missing comma at the end of this line and the subsequent attempts at creating primary keys on each table will cause these statements to fail.